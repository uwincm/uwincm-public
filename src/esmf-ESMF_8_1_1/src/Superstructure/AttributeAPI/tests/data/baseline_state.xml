<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated with ESMF Version 5.3.0, http://www.earthsystemmodeling.org -->
  <model_component name="N/A" full_name="N/A" version="N/A" 
      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" 
      xmlns:xlink="http://www.w3.org/1999/xlink" 
      xmlns:gco="http://www.isotc211.org/2005/gco" 
      xmlns:gmd="http://www.isotc211.org/2005/gmd" 
      xmlns="http://www.earthsystemmodeling.org" 
      xsi:schemaLocation="http://www.earthsystemmodeling.org file:/esmf_model_component.xsd">
    <variable_set>
    </variable_set>
  </model_component>
