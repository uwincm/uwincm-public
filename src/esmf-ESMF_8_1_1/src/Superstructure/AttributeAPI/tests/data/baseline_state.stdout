Name: state	  Convention: customconvention2	  Purpose: custompurpose2	


